xapian-letor/tests/api_letor.cc

```diff
@@ -43,6 +43,7 @@ db_index_one_document(Xapian::WritableDatabase& db, const string&)
     termgenerator.set_document(doc);
     termgenerator.set_stemmer(Xapian::Stem("en"));
     termgenerator.index_text("Tigers are solitary animals", 1, "S");
+    termgenerator.index_text("1", 1, "U/");
```

Comment:
`TermGenerator` is for generating terms by parsing text and splitting it into words.

For boolean terms, use `Document::add_boolean_term()`.

Reply:
How do I generate a URL extension (The U/"..") using `TermGenerator` other than this?